centreon/doc/API/latest/onPremise/Configuration/Service/DeployServices.yaml

```
@@ -1,9 +1,14 @@
+#/configuration/hosts/{host_id}/services/deploy:
 post:
   tags:
     - Service
   summary: "Add services to a host based on associated host template"
+  parameters:
+    - $ref: '../Host/QueryParameter/HostId.yaml'
   description: |
     Add services to a host based on associated host template
+  parameters:
```

yamllint (1.35.1)

[error] duplication of key "parameters" in mapping

(key-duplicates)